ngraph/core/include/ngraph/node.hpp

```diff
@@ -194,9 +198,20 @@ namespace ngraph
         /// \returns the autobroadcasr spec
         virtual const op::AutoBroadcastSpec& get_autob() const;
         /// \brief Evaluates the op on input_values putting results in output_values
+        /// \param outputs Tensors for the outputs to compute. One for each result
```

Comment:
minor: should it be "output_values", like in function declaration?

Reply:
done